### PR TITLE
Return 200 OK instead of 204 No Content

### PR DESCRIPTION
**Pull request by velichkoStoev**

### CODE REVIEW

**Review by evgeni-spasov-metrilo — COMMENTED** [review]
There is a text about 200 in the README.md.

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ curl -X POST "https://trk.mtrl.me/customer" -i -H "Content-Type: text/plain" -d
   }
 }'
 ```
-(We include the `-i` param so that you can verify a 204 status code for all these requests)
+(We include the `-i` param so that you can verify a 200 status code for all these requests)
```
Comment (evgeni-spasov-metrilo) [inline README.md:72]: @vgochev Does one need the `-i` param though?

Comment (vgochev) [reply README.md:72]: Not anymore - I think `curl` by default would return a 200 - should check with the API, though

Comment (vgochev) [reply README.md:72]: @velichkoStoev Oh, Java Master - would `curl` show 200 without the `-i` param when calling the updated API?

Comment (velichkoStoev) [reply README.md:72]: No:

```
➜  curl -X POST "https://tracking-mayna.customerboard.co/order" -H "Content-Type: text/plain" -d "{\"params\":{\"id\":441,\"createdAt\":1591362038333,\"email\":\"maina@jega.com\",\"amount\":63.92,\"coupons\":[],\"status\":\"processing\",\"products\":[{\"productId\":\"1322313123\",\"quantity\":\"1\"}],\"billing\":{\"firstName\":\"Scot \",\"lastName\":\"Taylor \",\"address\":\"5746 Swift Creek Rd ,\",\"city\":\"West Jordan \",\"countryCode\":\"US\",\"phone\":\"8016478611\",\"postcode\":\"84081\",\"paymentMethod\":\"Checkout With PayPal\"}},\"token\":\"1111111111111111\",\"platform\":\"Magento Community 2.3.5-p1\",\"pluginVersion\":\"2.0.0\",\"time\":1591362038333}"  
➜  curl -i -X POST "https://tracking-mayna.customerboard.co/order" -H "Content-Type: text/plain" -d "{\"params\":{\"id\":441,\"createdAt\":1591362038333,\"email\":\"maina@jega.com\",\"amount\":63.92,\"coupons\":[],\"status\":\"processing\",\"products\":[{\"productId\":\"1322313123\",\"quantity\":\"1\"}],\"billing\":{\"firstName\":\"Scot \",\"lastName\":\"Taylor \",\"address\":\"5746 Swift Creek Rd ,\",\"city\":\"West Jordan \",\"countryCode\":\"US\",\"phone\":\"8016478611\",\"postcode\":\"84081\",\"paymentMethod\":\"Checkout With PayPal\"}},\"token\":\"1111111111111111\",\"platform\":\"Magento Community 2.3.5-p1\",\"pluginVersion\":\"2.0.0\",\"time\":1591362038333}" 
HTTP/1.1 200 OK
Server: nginx/1.13.12
Date: Tue, 16 Jun 2020 12:18:03 GMT
Content-Length: 0
Connection: keep-alive
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=15724800; includeSubDomains
```

Comment (vgochev) [reply README.md:72]: ...I vote leave the `-i`?

**Review by evgeni-spasov-metrilo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (velichkoStoev) [reply README.md:72]: Yes, let's keep it, does no harm.

**Pull request merged**